application/src/main/java/org/togetherjava/tjbot/commands/free/UserStrings.java

```diff
@@ -23,8 +23,8 @@
             Command not ready please try again in a minute.
             """),
     NOT_MONITORED_ERROR("This channel is not being monitored for free/busy status. If you"
-            + "believe this channel should be part of the free/busy status system, please discuss it"
-            + "with a moderator"),
+            + " believe this channel should be part of the free/busy status system, please discuss it"
```

Comment:
Maybe change this to `, please consult a moderator`?

Reply:
Done, please check.